ngx-tools/src/utilities/object-deep-setter.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Helper function to get an object with deep keys
+ *
+ * @param obj - The object to modify.
+ * @param keys - The path of the property to set.
+ * @param value - The value to set.
+ * @return object
+ */
+
+export function objectDeepSet
+<ValueType, ObjectType extends object>(keys: string, value: ValueType, obj: any): ObjectType {
```

Comment:
More of a preference so feel free to offer a different viewpoint.. but offhand, I prefer the order of params like the JSDoc has them rather than the function itself. (obj, keys, value)